tests/phpunit/tests/Story_Renderer/HTML.php

```diff
@@ -41,12 +41,12 @@ public function tearDown() {
 	public function test_render() {
 		$post = self::factory()->post->create_and_get(
 			[
+				'post_type'    => \Google\Web_Stories\Story_Post_Type::POST_TYPE_SLUG,
 				'post_content' => '<!DOCTYPE html><html><head></head><body><amp-story></amp-story></body></html>',
 			]
 		);
```

Comment:
Would it help speed up tests if we just use `new Story([...])` in this file everywhere instead of creating new posts all the time?

Reply:
You were one who created these tests. Each post is different, so I dont see the issue.

Reply:
When I created the tests we didn't have the `Story` class 😉 

Doing `self::factory()->post->create_and_get()` X times is surely slower than doing `new Stoy()` X times. That's what I mean.

Reply:
Oh, I see. It doesn't need to be a post, is what you saying. I will change that.

Reply:
This is a more realistic test, passing the post object to the story object. I don't hate it as it is.